security/aporeto/build/ansible/group_vars/prod.yml

```diff
@@ -52,6 +52,7 @@ openshift_project_exclusions:
 - openshift-monitoring
 - "{{ openshift_project_prefix }}-operator"
 - "{{ openshift_project_prefix }}-enforcers"
+apb_external_nfs_host: ociopf-p-150.dmz
```

Comment:
Would it be better to use an IP here? or does this cause a lot of DNS lookups by using a hostname?

Reply:
I prefer using DNS names when possible as it generally helps with improve documentation and code readability, however, are we noticing a lot of DNS lookups from the enforcers? is this an issue?

Reply:
meh, should be fine. We can revisit if we find an issue.